### PR TITLE
Tweaks for chain and hacktheorb.explorer.agoric.net

### PR DESCRIPTION
**Pull request by michaelfig**
Just some scripting and config changes.  Rubber stamp appreciated.

### CODE REVIEW

File: packages/dapp-svelte-wallet/ui/src/Transaction.svelte [hunk 1]
```diff
@@ -114,7 +114,7 @@
     {/if}
     {#if status === 'proposed'}
     <div class="flex flex-row">
-    <Chip on:click={() => E(walletP).acceptOffer(offerId).then(showOutcome)}
+    <Chip on:click={() => E(walletP).acceptOffer(offerId)}
```
Comment (warner) [inline packages/dapp-svelte-wallet/ui/src/Transaction.svelte]: Is there some other pathway to displaying the success/failure of `acceptOffer`? (I have no idea how this works, just surprised to see a removal without an addition elsewhere)

**Review by warner — APPROVED** [review]
looks good, as far as I can tell

File: packages/dapp-svelte-wallet/ui/src/Transaction.svelte (hunk 1 above)
Comment (michaelfig) [reply packages/dapp-svelte-wallet/ui/src/Transaction.svelte]: It would be good to display a message on rejection.  I'll add that.

Comment (michaelfig) [reply packages/dapp-svelte-wallet/ui/src/Transaction.svelte]: (Success doesn't need a display, since now the Dapps do it themselves.)

**Pull request merged**